### PR TITLE
Combine email check with place order

### LINKED ISSUE #244: Combine Order info input and email input to one form

### PR DESCRIPTION
**Pull request by zachpren0**
close #244 

- removed match email function, email entry is now on order page
- functionality is there, need to work now on styling

### CODE REVIEW

File: MERN Scripts/backend/routes/api/charge.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 const express = require('express');
 const router = express.Router();
-const squareConnect = require('square-connect');
+const squareConnect = require('square');
 //const uuidv4 = require('uuid/v4');
-import { randomUUID } from 'crypto';
+
+
```
Comment (zachpren0) [inline MERN Scripts/backend/routes/api/charge.js:6]: changes to get server to start properly

File: MERN Scripts/mern-emsystem/src/components/hooks/placeOrder.js [hunk 2]
```diff
@@ -35,75 +90,128 @@ const PlaceOrder = () => {
         let orderVal = document.getElementById("quantity").value;
         if(orderVal >= 1 && orderVal <= 20 && (orderVal%1===0)){
             
+
```
Comment (zachpren0) [inline MERN Scripts/mern-emsystem/src/components/hooks/placeOrder.js:93]: handleSubmit() validates that email is proper using regex

File: MERN Scripts/mern-emsystem/src/components/hooks/placeOrder.js [hunk 3]
```diff
@@ -35,75 +90,128 @@ const PlaceOrder = () => {
...
         }
     }
 
-    const email = localStorage.getItem("email");
+    
 
     return (
-        <div>
-            <div className="loggedIn">
-                <p id="email" >Logged in as: {email} </p>
```
Comment (zachpren0) [inline MERN Scripts/mern-emsystem/src/components/hooks/placeOrder.js:54]: delete logged in as line

**Review by amurph24 — APPROVED** [review]
Nice work!

**Review by SpeiserK — APPROVED** [review]
Nice work combining the email check and place order page

**Review by camwlsn — APPROVED** [review]
Create job collating these pages

**Pull request merged**